medusa/databases/main_db.py

```diff
@@ -260,6 +262,7 @@ def fix_unaired_episodes(self):
                                    [common.UNAIRED, cur_unaired["episode_id"]])
 
     def fix_tvrage_show_statues(self):
+        #  Remove?
```

Comment:
No this one can stay. Still need to check all the show status we currently have. As I didn't sanitize these. Then we just rename this one.